lint-staged.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
 		"eslint --fix --cache --no-error-on-unmatched-pattern --quiet"
 	],
 	"package.json": (files) => [
-		"yarn constraints --fix && yarn install",
+		"`yarn constraints --fix && yarn install`",
```

Comment:
```suggestion
		"yarn constraints --fix",
		"yarn install",
```

@cdransf Oops! Your approach was right in the first place - lint-staged isn't running these as shell commands so the ampers won't work at all. 🥳

Reply:
Ohh gotcha! That makes sense. ✨